resources/views/admin/dashboard/index.blade.php

```diff
@@ -1,4 +1,4 @@
-@extends('admin.main')
+@extends('admin.dashboard.main')
```

Comment:
???
Why to create separate layout for dashboard?
Dashboard is the first page in admin. It's the only one page, not the entire admin module.